src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp

```diff
@@ -307,6 +329,108 @@ void ComputeNormalDotFluxes<Dim>::apply(
     }
   }
 }
+
+template <size_t Dim>
+void UpwindFlux<Dim>::package_data(
+    gsl::not_null<Variables<package_tags>*> packaged_data,
+    const typename Tags::UPsi<Dim, Frame::Inertial>::type& u_psi,
+    const typename Tags::UZero<Dim, Frame::Inertial>::type& u_zero,
+    const typename Tags::UPlus<Dim, Frame::Inertial>::type& u_plus,
+    const typename Tags::UMinus<Dim, Frame::Inertial>::type& u_minus,
+    const typename ::Tags::CharSpeed<Tags::UPsi<Dim, Frame::Inertial>>::type&
+        char_speed_u_psi,
+    const typename ::Tags::CharSpeed<Tags::UZero<Dim, Frame::Inertial>>::type&
+        char_speed_u_zero,
+    const typename ::Tags::CharSpeed<Tags::UPlus<Dim, Frame::Inertial>>::type&
+        char_speed_u_plus,
+    const typename ::Tags::CharSpeed<Tags::UMinus<Dim, Frame::Inertial>>::type&
+        char_speed_u_minus,
+    const typename Tags::ConstraintGamma2::type& gamma2,
+    const tnsr::i<DataVector, Dim, Frame::Inertial>& interface_unit_normal)
+    const noexcept {
+  get<Tags::UPsi<Dim, Frame::Inertial>>(*packaged_data) = u_psi;
+  get<Tags::UZero<Dim, Frame::Inertial>>(*packaged_data) = u_zero;
+  get<Tags::UPlus<Dim, Frame::Inertial>>(*packaged_data) = u_plus;
+  get<Tags::UMinus<Dim, Frame::Inertial>>(*packaged_data) = u_minus;
+  get<::Tags::CharSpeed<Tags::UPsi<Dim, Frame::Inertial>>>(*packaged_data) =
+      char_speed_u_psi;
+  get<::Tags::CharSpeed<Tags::UZero<Dim, Frame::Inertial>>>(*packaged_data) =
+      char_speed_u_zero;
+  get<::Tags::CharSpeed<Tags::UPlus<Dim, Frame::Inertial>>>(*packaged_data) =
+      char_speed_u_plus;
+  get<::Tags::CharSpeed<Tags::UMinus<Dim, Frame::Inertial>>>(*packaged_data) =
+      char_speed_u_minus;
+  get<Tags::ConstraintGamma2>(*packaged_data) = gamma2;
+  get<Tags::UnitNormalOneForm<Dim, Frame::Inertial>>(*packaged_data) =
```

Comment:
In the other fluxes we contract with the unit normal here in `package_data` already, then store the result in a bunch of tags that are private to the flux struct and retrieve them in `operator()`. Can you also do this here?